site/content/api/Command-And-Control-API.md

```diff
@@ -40,33 +39,24 @@ The following table provides an overview of the properties the *Business Applica
 | :--- | :-------: | :------- | :-------- | :---------- |
 | *message-id* | yes | *properties* | *string* | MUST contain the message ID used to match incoming control message with the response message containing result of execution. |
 | *subject* | yes | *properties* | *string* | MUST contain the command to be executed by a device. |
-| *content-type* | no | *properties* | *symbol*  | SHOULD be set to *application/octet-stream* if the message payload is to be considered *opaque* binary data. In most cases though, the client should be able to set a more specific content type indicating the type and characteristics of the data contained in the payload, e.g. `text/plain; charset="utf-8"` for a text message or `application/json` etc. |
-| *ttl* | no | *properties* | *int* | Indicates the timeout value (in milliseconds) indicating when server can assume that command has not been executed because of the lack of the response from the client. |
+| *content-type* | no | *properties* | *symbol*  | SHOULD be set to *application/octet-stream* if the message payload is to be considered *opaque* binary data. In most cases though, the sender should be able to set a more specific content type indicating the type and characteristics of the data contained in the payload, e.g. `text/plain; charset="utf-8"` for a text message or `application/json` etc. |
```

Comment:
I have the same question as for *ttl*: is this information conveyed to the device?